python/GafferUI/VectorDataPlugValueWidget.py

```diff
@@ -67,26 +84,37 @@ def _updateFromPlug( self ) :
 		plug = self.getPlug()
 		if plug is not None :
 			with self.getContext() :
-				plugValue = plug.getValue()
-				if plugValue is None :
-					# the VectorDataWidget isn't so keen on not having data to work with,
-					# so we'll make an empty data of the right type.
-					plugValue = plug.ValueType()
-				self.__dataWidget.setData( plugValue )
+				self.__dataWidget.setData( [ p.getValue() for p in self.__dataPlugs() ] )
 
 			dragPointer = Gaffer.Metadata.value( plug, "vectorDataPlugValueWidget:dragPointer" )
 			if dragPointer is not None :
 				self.__dataWidget.setDragPointer( dragPointer )
 
 		self.__dataWidget.setEditable( self._editable() )
 
+	def __dataPlugs( self ) :
+
+		if not len( self.getPlug() ) :
```

Comment:
I think it would be fairly surprising for the singular `getPlug()` to return a list. I could change this to `len( self.getPlug().children() )`, but it would be wordier and would be building a temporary python list we don't need.

Reply:
Yeah, as I said, probably not anything wrong with the code.  Just a place where the Pythonic way of doing things makes me uncomfortable.